bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcPanel.scala

```diff
@@ -403,3 +445,37 @@ protected class PlotDetailsPanel extends GridBagPanel {
 
 }
 
+// light weight wrapper to turn NumberBoxWidget into a Scala swing component
+private class NumberEdit(label: Label, units: Label, default: Double = 0) extends Component {
+  override lazy val peer = new NumberBoxWidget {
+    setColumns(8)
+    setValue(default)
+    setMinimumSize(getPreferredSize)
+    addWatcher(new TextBoxWidgetWatcher {
+      override def textBoxKeyPress(tbwe: TextBoxWidget): Unit = textBoxAction(tbwe)
+      override def textBoxAction(tbwe: TextBoxWidget): Unit =
+        try {
+          publish(new ValueChanged(NumberEdit.this))
+          tbwe.requestFocus()
+        }
+        catch {
+          case _: NumberFormatException =>
+        }
```

Comment:
I realize this is insanely picky but the newline before `catch` is making me crazy.